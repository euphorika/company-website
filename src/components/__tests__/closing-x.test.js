import React from "react"
import { shallow } from "enzyme"
import toJson from 'enzyme-to-json'

import ClosingX from "../closing-x"

describe("ClosingX", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<ClosingX color="#fff" />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
