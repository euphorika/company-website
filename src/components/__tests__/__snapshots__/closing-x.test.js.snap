// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClosingX renders correctly 1`] = `
<svg
  viewBox="0 0 172.26 174.13"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M172.26 28.71L143.55 0 86.13 57.42 28.71 0 0 28.71l57.42 57.42L0 143.54l28.71 28.72 57.42-57.42 57.42 57.42 28.7-28.72-57.41-57.41z"
    fill="#fff"
  />
</svg>
`;
